cpu/cc2538/dev/cc2538-ccm-star.c

```diff
@@ -75,49 +75,18 @@ restore_crypto(uint8_t enabled)
 static void
 set_key(const uint8_t *key)
 {
-  cc2538_aes_128_driver.set_key(key);
+  ccm_star_driver.set_key(key);
```

Comment:
Thanks. I'll look into it. In the meantime, if there is an urgent need to fix this, your commit can be used, then reverted once a proper fix using the hardware acceleration has been found.

I have a few questions:
 - With which board have you tested this?
 - Do both the sender and the receiver crash, or only one of them?
 - Why do I get the following errors with a clean checkout of your branch, if building for Zoul Firefly?

Sender:
```
obj_zoul/netstack.o: In function `netstack_init':
netstack.c:(.text.netstack_init+0x30): undefined reference to `adaptivesec_driver'
obj_zoul/secmac.o: In function `on_fifop':
secmac.c:(.text.on_fifop+0x66): undefined reference to `potr_parse_and_validate'
obj_zoul/secmac.o: In function `on_rxpktdone':
secmac.c:(.text.on_rxpktdone+0x16): undefined reference to `potr_shall_send_ack'
obj_zoul/secmac.o: In function `process_thread_post_processing':
secmac.c:(.text.process_thread_post_processing+0x160): undefined reference to `adaptivesec_framer'
secmac.c:(.text.process_thread_post_processing+0x168): undefined reference to `adaptivesec_driver'
sender.co: In function `process_thread_sender_process':
sender.c:(.text.process_thread_sender_process+0x2c): undefined reference to `neighbor_new'
sender.c:(.text.process_thread_sender_process+0x98): undefined reference to `adaptivesec_group_key'
sender.c:(.text.process_thread_sender_process+0xa4): undefined reference to `adaptivesec_driver'
```

Receiver:
```
obj_zoul/netstack.o: In function `netstack_init':
netstack.c:(.text.netstack_init+0x30): undefined reference to `adaptivesec_driver'
obj_zoul/secmac.o: In function `on_fifop':
secmac.c:(.text.on_fifop+0x66): undefined reference to `potr_parse_and_validate'
obj_zoul/secmac.o: In function `on_rxpktdone':
secmac.c:(.text.on_rxpktdone+0x16): undefined reference to `potr_shall_send_ack'
obj_zoul/secmac.o: In function `process_thread_post_processing':
secmac.c:(.text.process_thread_post_processing+0x160): undefined reference to `adaptivesec_framer'
secmac.c:(.text.process_thread_post_processing+0x168): undefined reference to `adaptivesec_driver'
receiver.co: In function `process_thread_receiver_process':
receiver.c:(.text.process_thread_receiver_process+0x20): undefined reference to `neighbor_new'
receiver.c:(.text.process_thread_receiver_process+0x48): undefined reference to `adaptivesec_group_key'
```